data/remnant/remnant 2 side missions.txt

```diff
@@ -640,3 +640,59 @@ mission "Remnant: Shattered Light 6"
+	landing
+	source
+		government "Remnant"
+	destination "Caelian"
+	to offer
+		has "Remnant: Cognizance 25: done"
+	npc "scan outfits"
+		government "Korath"
+		personality waiting target
+		system
+			system "Kor Zena'i"
+		ship "Tubfalet" "Ra'puk At 295"
+		dialog `In poor lighting this ship might have passed for a Palavret, but major structural difference suggest this is at least a different model, or perhaps an entirely different ship. Time to report this to Chilia.`
+	npc
+		government "Korath"
+		personality waiting
+		system
+			system "Kor Zena'i"
+		fleet
+			names "korath"
+			cargo 3
+			commodities Food Clothing Metal Plastic Medical "Heavy Metals"
+			outfitters "Korath Basics" "Korath Exiles"
+			variant
+				"'olofez" 3
+	on offer
+		event "remnant: tubfalet appearance" 20 31
+		conversation
+			`You step out of your airlock to see Prefect Chilia striding towards you. "Captain <first>! I read a report mentioning that you dealt with a trio of 'olofez out in the Postverta region. Would you be interested in doing some reconnaissance for us? We would like to track down where those came from."`
+			choice
+				`	"Yes, I could do that."`
+				`	"I would rather not."`
+					decline
+			`	He signs appreciatively. "We appreciate your willingness to help." He pulls up a hologram of the Ember Waste, and it shifts to display the systems towards the core. "Based on the Korath's typical movement patterns, we suspect that the ship should be somewhere in the nearer Korath systems. We will upload an algorithm to your targeting system to look for ship profiles that have no matches in our database. Please do an outfit scan of whatever ship you find."`
+				accept
+	on enter "Kor Zena'i"
+		dialog
+			`	The algorithm quickly parses the ship signatures present here, and notifies you that one in particular does not match its database: the one identified as the Ra'puk At 295. This would be the ship that Chilia would like a scan of.`
+	on complete
+		payment 150000
+		conversation
+			`Chilia is waiting for you when you land. He looks over the scan results, nodding thoughtfully, and listens to your account of visiting Kor Zena'i.`
+			`	"Thank you for investigating this, <first>. I will ensure that our engineers get your data immediately, and brief our patrols to be aware of this increased capacity." His gestures have a tone of anticipation and regret.`
+			choice
+				`	"You are welcome."`
+					goto scanningpayment
+				`	"Are you expecting something?"`
+			`	Chilia smiles wistfully. "Do you know that feeling, when the whole world is still and the skies are clear, and yet you know in your bones that a storm is coming?" He continues without waiting for a response. "Well, I have had that feeling for decades, growing ever stronger. And now..." He pauses, drawing a deep breath. "Now, it feels like this is the first scent of rain. The first thing I can actually identify as a solid indicator that a storm is brewing."`
```

Comment:
You feel the air is [charged, tense] like the moment before a storm? Seems that's going around.
Though, honestly, if you've had that feeling for decades, you're probably just paranoid. Even if the "storm" does come, at this point, it's just a broken clock being right twice a day. You should speak to professional, Chilia.

Reply:
hehe. Yeah, probably...